src/general/cpuInfo.go

```diff
@@ -47,30 +47,54 @@ func NewCPULoad() *CPULoad {
 	return &CPULoad{}
 }
 
+// ReadCPULoad reads /proc/stat and returns the average load
+func readCPULoad(c *CPULoad) error {
+	data, error := os.ReadFile("/proc/stat")
```

Comment:
`error` is a type name as well. Maybe rename this to something like `err`?

Reply:
That's the return type, so it would be a typename right? UpdateCPULoad and GetCPULoad also use `error`

Reply:
Oh no that's not what I meant, I meant changing `data, error := ...` to `data, err := ...`

Reply:
Ahhh nevermind lol